js/layout-javascript/small-card-code.js

```diff
@@ -124,6 +124,11 @@ DynamicList.prototype.attachObservers = function() {
     }
   });
 
+  _this.Utils.Hooks.activeFilters({
```

Comment:
```suggestion
  if (_this.data.filtersInOverlay) {
    Fliplet.Hooks.on('flListDataAfterRenderList', function toggleActiveFilters() {
      _this.Utils.UI.showActiveOverlayFilters();
    });
  }
```

Reply:
@tonytlwu Do I get it right that we need to show active filters only when filter overlay is active?

Reply:
@YaroslavOvdii Yes. The proposed feature said, "Show active filters when users apply filters **via the overlay**."